.gitmodules

```diff
@@ -1,4 +1,8 @@
-[submodule "packages/core-sdk/protocol-contracts"]
```

Comment:
Since we have the wagmi generator, this part is not really needed I think. But it can be used as a backup when wagmi generator doesn't work. Just FYI